components/basecomponents/Navigation/Dropdown/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
+import clsx from 'clsx';
+import Link from 'next/link';
+import { getCountryVariant } from 'utils/locales';
+import { SubmenuItem } from 'data/headerNavigation';
+import { OpenDropdown } from '..';
+
+interface DropdownItems extends SubmenuItem {
+  active?: boolean;
+}
+
+interface DropdownProps {
+  submenu: SubmenuItem[];
+  dropdown: OpenDropdown;
+  id: string;
+  callback: (val: boolean) => void;
+}
+
+export const Dropdown: React.FC<DropdownProps> = ({ submenu, dropdown, id, callback }) => {
+  const { t } = useTranslation('common');
+  const { asPath } = useRouter();
+  const visible = dropdown.id === id;
+
+  const items: DropdownItems[] = submenu
+    .filter((item) => item.countryVariant.includes(getCountryVariant()))
+    .map((item) => {
+      const { link } = item;
+
+      if (link === asPath) {
+        return {
+          ...item,
+          active: true,
+        };
+      }
+
+      return { ...item };
+    });
+
+  return (
+    <div className="md:absolute md:top-12">
+      <ul className={clsx({ [styles.basic]: true, hidden: !visible || !dropdown.open })}>
+        {items.map(({ name, link, active }, index) => {
+          return (
+            <li
+              key={index}
+              id={id}
+              className={clsx({
+                'text-primary-yellow md:text-gray-700 md:list-disc': active,
+                'hover:text-primary-blue my-2 md:my-1 text-sm': true,
```

Comment:
The links inside dropdown should be clickable in their entire area, not just their text :)